Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 
+ENV test_file="core/Akka.Streams.Tests/Akka.Streams.Tests.csproj"
+ENV test_filter=""
+ENV run_count=2
+RUN mkdir akka
+COPY . ./akka
+RUN ls
+WORKDIR /akka/src
+CMD ["/bin/bash", "-c", "x=0; while [ $x -le ${run_count} ]; do dotnet test ${test_file} ${test_filter} --framework net6.0 --logger trx;  x=$(( $x + 1 )); sleep 5s; done"]
```

Comment:
I would design this to basically run until the test fails, rather than keep running even after it's failed. That will reduce the amount of time people have to wait, reduce the logs, and tighten the feedback loop to developers.

You'll know if the test failed based on the exit code returned by `dotnet test`